src/index.html

```diff
@@ -1,13 +1,56 @@
 <!DOCTYPE html>
 <html lang="en-US">
   <head>
-    <title>Webpack Starter Kit</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Oxygen:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/74176d5c0d.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <link rel="stylesheet" href="/src/css/style_two.css" />
+
+    <title>Travel Tracker</title>
   </head>
   <body>
-    <h1>Webpack Starter Kit</h1>
-
-    <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
+    <div id="site-container">
+      <div id="header-section">
+        <h1>Travel Tracker <i class="fas fa-passport"></i></h1>
+      </div>
+      <div id="main-section">
+        <form id="login-form">
+          <label id="signin" for="signin">Sign In</label><br />
+          <div id="username">
+            <label id="label-username" for="username">Username:</label><br />
+            <input
+              id="input-username"
+              type="text"
+              placeholder="Please enter your username"
+            /><br />
+          </div>
+          <div id="password">
+            <label id="label-password" for="password">Password:</label><br />
+            <input
+              id="input-password"
+              type="text"
+              placeholder="Please enter your password"
+            /><br />
+          </div>
+          <button type="button" onclick="authPW()" id="submit">SUBMIT</button>
```

Comment:
I'd like for us to figure out how to add this event listener somewhere else beside the html element.  essentially lets get an onclick in a javascript file that selects that button.  We don't need to call JS in our html files.  It works, but lets clean her up :)